### DISCUSSION TITLE
We transcribed a Hindi audio file and found that the English words in output.

### DISCUSSION THREAD

**Discussion by mail1906** [deepgram/community#402] [Feature Requests]
We transcribed a Hindi audio file and found that the English words spoken in a Hindi sentence is output in English characters. Is it possible to output the English words also in Hindi characters (not translated)? Take the below sentence for example. 

**उसका तो tune थोड़ी ना change हो गया, वह तो वही prospecting वाली किताब पढ़ रहा था**
should ideally be 
**उसका तो ट्यून थोड़ी ना चेंज हो गया, वह तो वही प्रॉस्पेक्टिंग वाली किताब पढ़ रहा था**

Few words are wrongly transcribed.  For example, Jim is output as Gym. We tried adding Jim to keywords list and it did not make any difference. Is there a way to fix such mistakes?

**Comment by jkroll-deepgram** [comment]
Hi @mail1906, do you have a request ID, and/or the URL parameters you were using for that request? That will help me look further into how you're using Deepgram.

Whisper does sometimes apply translation in unexpected ways.

Deepgram's models do not translate, but if English words are intermixed with Hindi in the original audio, we will occasionally transcribe those words in English (and Latin script / English characters).

None of the models have the ability to force all transcription to Devanagari / Hindi characters.

**Comment by ankitpanchotiya99138** [comment]
On web socket how can I add translation do we have any particular method or link ? @jkroll-deepgram

**Reply by jkroll-deepgram** [reply]
Hi @ankitpanchotiya99138, Deepgram's own models do not offer translation. With our managed Whisper model, you can provide your non-English audio with the parameter `language=en`, which will prompt Whisper to translate the audio into English.

**Comment by manav741** [comment]
Hii @jkroll-deepgram , while i am using the playground i am getting correct output but as soon as i start making a hit with my api i only get the english characters used in that audio in my transcripts.
I haven't changed any parameters even.
        const deepgramApiUrl = 'https://api.deepgram.com/v1/listen';
        const deepgramApiData = {
            model: 'general',
            tier: 'enhanced ',
            version: 'latest',
            language: 'hi',
            diarize: true,
            smart_format: true,
            url: audioUrl,
        };
        const deepgramApiHeaders = {
            accept: 'application/json',
            'content-type': 'application/json',
            why is this happening?

**Reply by jkroll-deepgram** [reply]
Hi @manav741, I just did a quick test on some Hindi audio using your parameters, and that did return Devanagari characters with only a few Latin characters showing up when English words were spoken.

If you're still troubleshooting this issue, can you share a request ID of yours? That will enable me to look into it further for you. It's returned in the response metadata (`response["metadata"]["request_id"]`).

**Comment by manav741** [comment]
It is working now.

On Thu, Dec 28, 2023 at 2:42 AM Julia Kroll ***@***.***>
wrote:

> Hi @manav741 <https://github.com/manav741>, I just did a quick test on
> some Hindi audio using your parameters, and that did return Devanagari
> characters with only a few Latin characters showing up when English words
> were spoken.
>
> If you're still troubleshooting this issue, can you share a request ID of
> yours? That will enable me to look into it further for you. It's returned
> in the response metadata (response["metadata"]["request_id"]).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/deepgram/discussions/402#discussioncomment-7960190>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BEZOBGLNJE4T27ESWTMIXGTYLSFKPAVCNFSM6AAAAAA6XT2A36VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TSNRQGE4TA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 


-- 
This message (including any attachment(s) hereto) is confidential and may 
also be privileged. It is intended solely for the addressee. If you are not 
the intended recipient you are hereby notified that any disclosure, 
copying, distribution or taking any action in reliance on the contents of 
this information is strictly prohibited and may be unlawful. If you have 
received this message in error you are requested to delete it from your 
system and contact the sender by replying to this message immediately. 
Indifi Technologies Private Limited is not liable for the improper 
transmission of this message nor for any damage sustained as a result of 
this message.
